symphony-bdk-core/src/test/java/com/symphony/bdk/core/util/PresentationMLParserTest.java

```diff
@@ -34,6 +34,14 @@ void getMessageFromPresentationMLNotTrimTest() throws PresentationMLParserExcept
     assertEquals(content.trim(), "This is a link to Symphony's Website");
   }
 
+  @Test
+  void getMessageFromPresentationMLWithEscapedContentTest() throws PresentationMLParserException {
+    String presentationML = "<div data-format=\"PresentationML\" data-version=\"2.0\">&nbsp;</div>";
+    String content = PresentationMLParser.getTextContent(presentationML, false);
+
+    assertNotEquals(content, "&#160;");
```

Comment:
and what should it be equals to? :) " " (that would be more restrictive)

Reply:
my mistake! Let me fix it!